docker-compose.yml

```diff
@@ -28,9 +29,8 @@ services:
       bash -c "
       cp /app/certs/rootCA.pem /usr/local/share/ca-certificates/rootCA.crt && \
       update-ca-certificates && \
-      uv run flask db upgrade
-
-      (uv run flask vite install && uv run flask vite start) &
+      npm run dev -- --host & \
+      uv run flask db upgrade & \
```

Comment:
Not sure if the reordering is intentional here? I think I still would like cp/update/db upgrade to be a single group that runs first and if it fails blocks everything, then npm run dev/python run server as two parallel processes? Is that a bad idea? 🤔

Reply:
Yeah good shout, I think this was just a slightly careless rebase that didn't catch the fix on main